invenio_banners/resources/config.py

```diff
@@ -18,7 +19,20 @@ class BannerResourceConfig(RecordResourceConfig):
     # Blueprint configuration
     blueprint_name = "banners"
     url_prefix = "/banners"
-    routes = {"create": "/new"}
+    routes = {
+        "create": "/new",
+        "banner": "/<banner_id>",
```

Comment:
```suggestion
        "item": "/<banner_id>",
```
to keep the consistency with the rest of the module